<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginScreen</class>
 <widget class="QWidget" name="LoginScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>280</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="login">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>登录/注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gameButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rankinglistButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>500</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>排行榜</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>194</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
